project_script/ramp_time_treatment.py

```diff
@@ -108,28 +93,31 @@ def integrate_data():
 #     decimal=',',
 #     )
 
-odd_columns_subtracted_smoothed_transposed = np.array(dt.smooth(odd_columns_subtracted_transposed_in_arrays))
+odd_columns_subtracted_smoothed = np.array(dt.smooth_current_noise(odd_columns_subtracted_transposed_in_arrays))
 
-peaks = dt.find_the_peaks(odd_columns_subtracted_smoothed_transposed)
+current_peaks = dt.find_peaks(odd_columns_subtracted_smoothed)
 
-indexes_list = dt.find_peak_index(odd_columns_subtracted_smoothed_transposed, peaks)
+indexes_list = dt.find_peak_index(odd_columns_subtracted_smoothed, current_peaks)
 
-time_max_values = dt.flatten(dt.find_time_max(time_photo_celiv_ramp_time_transposed, indexes_list))
+time_max_values = dt.flatten_list_of_lists(dt.find_time_max(time_photo_celiv_ramp_time_transposed, indexes_list))
 
-peak_values = pd.DataFrame([peaks, time_max_values]).transpose().set_axis(['current peak', 'peak time'], axis=1)
+peak_values = pd.DataFrame([current_peaks, time_max_values]).transpose().set_axis(['current peak', 'peak time'], axis=1)
 
 peak_values['first term of mobility calculations'] = cv.DEVICE_THICKNESS ** 2 / (2 * peak_values['peak time'] ** 2) \
     .round(decimals=30)
 
+last_current = pd.DataFrame(
+    current_dark_celiv_ramp_time.iloc[len(current_dark_celiv_ramp_time)-1, x]
+    for x in list(range(0, (len(current_dark_celiv_ramp_time.columns)))))
```

Comment:
As like the indentation above, this code doens't be like this?

![image](https://user-images.githubusercontent.com/11571434/159492872-ced86ff3-f55d-4c05-bd2f-5a8fd4b2caca.png)